build.gradle

```diff
@@ -54,6 +54,9 @@ ext {
     // The local git repository, typically in the .git directory, but not for worktrees.
     // This value is always overwritten, but Gradle needs the variable to be initialized.
     localRepo = ".git"
+
+    // Location of pom files used only to add meta-data to the artifacts published to Maven.
```

Comment:
Is the location used only for that purpose, or are the pom files used only for that purpose?
(Using a complete sentence could have prevented my question.)